src/Components/General/Navbar.test.js

```diff
@@ -0,0 +1,38 @@
+import { render } from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import Navbar from "./Navbar";
+
+test("check for Navbar class", () => {
+  const { container } = render(
+    <Router>
+      <Navbar />
+    </Router>
+  );
```

Comment:
you know when I was at airbnb we used to do it this way blah blah.

Reply:
testing libraries is a bit new thing for me, so I will add as you mentioned as of now.